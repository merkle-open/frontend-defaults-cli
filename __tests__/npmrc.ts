/// <reference types="@types/jest" />
import { apiIt } from './shared';

describe('npmrc', () => {
	it('default', async () =>
		await apiIt('npmrc', {
			npmrc: true,
		}));
});
