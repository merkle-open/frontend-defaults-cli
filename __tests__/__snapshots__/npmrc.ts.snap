// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`npmrc default 1`] = `
"{
  \\"options\\": {
    \\"ts\\": false,
    \\"tslint\\": false,
    \\"es\\": false,
    \\"eslint\\": false,
    \\"editorconfig\\": false,
    \\"prettier\\": false,
    \\"stylelint\\": false,
    \\"gitignore\\": false,
    \\"npmrc\\": true,
    \\"readme\\": false,
    \\"githooks\\": false,
    \\"commitlint\\": false,
    \\"nodeVersion\\": false,
    \\"webpack\\": false,
    \\"build\\": false,
    \\"install\\": false,
    \\"force\\": true,
    \\"dryRun\\": false,
    \\"mode\\": \\"api\\"
  },
  \\"mergedFiles\\": {
    \\".npmrc\\": {
      \\"override\\": false,
      \\"data\\": \\"save-exact=true\\\\n\\"
    }
  }
}"
`;

exports[`npmrc default 2`] = `
"save-exact=true
"
`;
